test/tests/testSendNotification.js

```diff
@@ -170,6 +170,8 @@ suite('sendNotification', function() {
     .then(function() {
       assert(false, 'sendNotification promise resolved');
     }, function() {
```

Comment:
Right, they're quite cryptic (I've had to use a lot of hacks to write them). I've filed #67 to write the documentation.